src/oktaSignInWidget/TermsAndConditionsDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogContentText from "@mui/material/DialogContentText";
+import DialogTitle from "@mui/material/DialogTitle";
+
+const TermsAndConditionsDialog = (props) => {
+  return (
+    <Dialog
+      maxWidth="md"
+      open={props.open}
+      keepMounted
+      onClose={() => props.closeTermsAndConditionsModal}
+      onBackdropClick={props.closeTermsAndConditionsModal}
+      aria-describedby="alert-dialog-slide-description"
+      data-testid={`terms-and-conditions-dialog-${
+        props.open ? "visible" : "hidden"
+      }`}
+    >
+      <DialogTitle>Terms and Conditions</DialogTitle>
+      <DialogContent>
+        <DialogContentText id="alert-dialog-slide-description">
+          OMB No.0938-1236 |{" "}
+          <a href="https://www.cms.gov/regulations-and-guidance/legislation/paperworkreductionactof1995">
+            {" "}
+            Paperwork Reduction Act
+          </a>
+          <br />
+          <br />
```

Comment:
if these `<br/>` are used to separate text into paragraphs and not line breaks, I would simply use `p` tag

Reply:
Updated it